### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by putaojuice**

### CODE REVIEW

**Review by yumengtan — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -127,7 +128,7 @@ Format: `findt KEYWORD [MORE_KEYWORDS]`
 - The order of the keywords matters. e.g. `drink water` will not match `water drink`
 - Only full keyword(s) will be accepted. e.g. `drink` will match `drink`; `ink` will not match `drink`
 - All keyword(s) have to be matched for task to be returned.
-- If a word is contained within a parantheses without spaces, the keyword has to contain the parantheses as well.
```
Comment (ivorcmx) [inline docs/UserGuide.md:130]: @yumengtan nice try with the spelling! 😵‍💫

Comment (ivorcmx) [reply docs/UserGuide.md:130]: @putaojuice great catch!

Comment (yumengtan) [reply docs/UserGuide.md:130]: > @yumengtan nice try with the spelling! 😵‍💫

oops! 🙊  @putaojuice thanks for catching my mistake! 💯

File: docs/UserGuide.md [hunk 2]
```diff
@@ -127,7 +128,7 @@ Format: `findt KEYWORD [MORE_KEYWORDS]`
 - The order of the keywords matters. e.g. `drink water` will not match `water drink`
 - Only full keyword(s) will be accepted. e.g. `drink` will match `drink`; `ink` will not match `drink`
 - All keyword(s) have to be matched for task to be returned.
-- If a word is contained within a parantheses without spaces, the keyword has to contain the parantheses as well. 
+- If a word is contained within a parentheses without spaces, the keyword has to contain the parantheses as well.
```
Comment (yumengtan) [inline docs/UserGuide.md:131]: The second occurence of parentheses still has a spelling mistake! Will be fixing this issue in next PR! 🔥

**Comment by putaojuice** [comment]
Fixes #132 #131

**Pull request merged**